python/ray/serve/tests/test_http_cancellation.py

```diff
@@ -139,10 +139,13 @@ async def __call__(self):
     except httpx.ReadTimeout:
         pass
 
-    wait_for_condition(
-        lambda: ray.get(collector.get.remote())
-        == ["Child_CancelledError", "Parent_CancelledError"]
-    )
+    def check_result():
+        result = ray.get(collector.get.remote())
+        return set(result) == {"Child_CancelledError", "Parent_CancelledError",} or set(
+            result
+        ) == {"Parent_RequestCancelledError", "Child_CancelledError"}
+
+    wait_for_condition(check_result, timeout=10)
```

Comment:
Hmm we don't expect `Parent_RequestCancelledError` to be thrown here though. Do you know why it sometimes gets thrown?

Reply:
this has something to do with anyscale grpc configs. see the two run below one with anyscale grpc configs which results in `Parent_RequestCancelledError` and one without  anyscale grpc configs which causes `Parent_CancelledError`
https://gist.github.com/abrarsheikh/f9bf0f622ed4bf14822df3bbd21ae291

Reply:
reverting this change, since it only effects turbo